### ISSUE TITLE
Compling error in Windows 8

### ISSUE THREAD

**Issue by EminThaqi** [kotas/gulp-tsc#4]
Hi there, 
I'm getting an error when I use your plugin in Windows 8. In Linux works fine though.

``` bash
[gulp] Failed to compile TypeScript: Error: Command failed: The filename, directory name, or volume label syntax is incorrect
```

**Comment by kotas** [comment]
@EminThaqi Thank you for your report!

I have just published v0.0.4 to fix problems on Windows.

I tested v0.0.4 on my Windows machine, but I'm not 100% sure that fixes your problem because of various node environment available on Windows. So could you try it and tell me if it still has issues. Thank you!

**Comment by EminThaqi** [comment]
Yup, thanks for quick reply, it works in:
Windows 8.1 Pro - x64 and elementaryOS (Ubuntu 12.04) x64.
Node environment x64: v0.10.25;
